control-plane-operator/hostedclusterconfigoperator/controllers/resources/kas/admissionpolicies.go

```diff
@@ -25,11 +27,12 @@ type AdmissionPolicy struct {
 }
 
 const (
-	AdmissionPolicyNameConfig = "config"
-	AdmissionPolicyNameMirror = "mirror"
-	AdmissionPolicyNameICSP   = "icsp"
-	AdmissionPolicyNameInfra  = "infra"
-	cnoSAUser                 = "system:serviceaccount:openshift-network-operator:cluster-network-operator"
+	AdmissionPolicyNameConfig          = "config"
+	AdmissionPolicyNameMirror          = "mirror"
+	AdmissionPolicyNameICSP            = "icsp"
+	AdmissionPolicyNameInfra           = "infra"
+	AdmissionPolicyNameMirroredConfigs = "mirroredconfigmaps"
```

Comment:
nit: can we prefix this and everything that is driven by NTO e.g. `AdmissionPolicyNameNTOMirroredConfigs = ntomirroredconfigmaps`?

Reply:
Done. Thanks!